ethcore/src/spec/spec.rs

```diff
@@ -240,7 +278,11 @@ fn load_from<T: AsRef<Path>>(cache_dir: T, s: ethjson::spec::Spec) -> Result<Spe
 		timestamp: g.timestamp,
 		extra_data: g.extra_data,
 		seal_rlp: seal_rlp,
-		constructors: s.accounts.constructors().into_iter().map(|(a, c)| (a.into(), c.into())).collect(),
+		constructors: s.accounts
+		               .constructors()
```

Comment:
[This is "smart tabs"](https://www.emacswiki.org/emacs/SmartTabs), it's just how it gets laid out automatically for me. I'll change it so the default is closer to our style.